base/src/main/java/com/smartdevicelink/proxy/rpc/OnVehicleData.java

```diff
@@ -458,11 +467,29 @@ public Double getInstantFuelConsumption() {
         return SdlDataTypeConverter.objectToDouble(object);
     }
 
+    /**
+     * Sets the externalTemperature.
+     *
+     * @param externalTemperature The external temperature in degrees celsius. This parameter is deprecated starting RPC
+     * Spec 7.1, please see climateData.
+     * @since SmartDeviceLink 2.0.0
+     * @deprecated in SmartDeviceLink 7.1.0
+     */
+    @Deprecated
     public OnVehicleData setExternalTemperature(Double externalTemperature) {

```

Comment:
@kostyaBoss On line 479 this should be `Float` instead of `Double`.

Reply:
@kostyaBoss Yes that makes sense.  This code has been there for a while, and the issue you mentioned is there.  You can just leave it alone.